src/i18n/en.json

```diff
@@ -1296,6 +1311,9 @@
     "motion.finalizeStep.submit": "Finalize",
     "motion.finalizeStep.transactions.remaining": "{transactions} transactions remaining",
     "motion.finalizeStep.claimed": "Claimed",
+    "motion.outcomeStep.win.title": "Congratulations! Your side won.",
+    "motion.outcomeStep.lost.title": "Sorry, your side lost.",
```

Comment:
I think these might have slipped up on the commit as I see that they aren't being used anywhere.

Reply:
These messages still need to be removed as they are not being used anywhere.